limits-requests/config.yaml

```diff
@@ -0,0 +1,15 @@
+code:
+  name: dkube-examples
+  url: https://github.com/oneconvergence/dkube-examples.git
+  branch: tensorflow
+  private: false
+  
+arguments:
+  code: dkube-examples
+  run_script: stress --vm-bytes 1000G
+  limit_cpu:  1000
+  limit_memory: 1000Gi
+  request_cpu: 999
+  request_memory: 999Gi
```

Comment:
these are too big limits and requests. Try with a reasonable limit and requests and then run_script should try to use more than limit and fail.

Reply:
Tried giving less values, 
Raised the issue https://github.com/oneconvergence/gpuaas/issues/7089